website/docs/r/cdn_frontdoor_secret.html.markdown

```diff
@@ -95,8 +95,6 @@ resource "azurerm_cdn_frontdoor_secret" "example" {
 
 ## Arguments Reference
 
-The following arguments are supported:
```

Comment:
why's this been removed/can we add this back in?

Reply:
Fixed.